### DISCUSSION TITLE
Would it be possible to pass any additional options to stylelint?

### DISCUSSION THREAD

**Discussion by ElMassimo** [amatlash/vite-plugin-stylelint#2] [Ideas]
Hi there, I was wondering if the current setup would allow passing the `--fix` flag as an option to Stylelint, so in addition of linting, it would auto-correct files.

I've used auto-fix with ESLint in the past, but I'm not sure if that's possible in Vite.

### Motivation 🤔 

I like to use `stylelint-order`, and I want the formatting to occur as I go, rather than all at once when making a commit.

**Comment by amatlash** [comment]
Hi, I`ve finally figured out my thoughts about your question.
I personally hate using auto-fixing for both linters due to loosing control feeling)

Technically, this flag can be passed and file could be auto-corrected and saved. That`s not a problem.

I cannot predict various IDEs behaviour in case of auto-correcting file, that has unsaved changes. I don't know if any IDE merges unsaved changes with new version of file. It's common to forget to save file (if you aren't using auto-save) while rapid development.

So I will add that feature with some warning after small research about that. I bet vue-cli plugins contributors have faced with same questions.

**Reply by ElMassimo** [reply]
Makes sense, as long as it's off by default, it shouldn't be a problem.

I brought it up because right now there's no way to pass in additional options to Stylelint (other than `include` and `exclude`). Renaming the discussion topic to clarify.

Ideally, any option that `stylelint` might receive should be passed over.
